index.html

```diff
@@ -257,7 +257,8 @@ <h3>CredentialSchema2023</h3>
                       <td>id</td>
                       <td>The constraints on the <code>id</code> property are listed in the Verifiable Credentials
                       Data Model specification [[VC-DATA-MODEL-2]]. The value is expected to be a URL that identifies
-                      the credential schema associated with the <a>verifiable credential</a>.</td>
+                      the <a>verifiable credential</a> which contains a credential schema. It is RECOMMENDED that
+                      the <code>id</code> of the verifiable credential match that of the credential schema.</td>
```

Comment:
I think we should remove this recommendation. A typical ID of a schema ends in `.json`, but a VC may be communicated in different formats (`vc+ld+json` or `vc+jwt` are examples).

Reply:
agreed